src/components/App/app.test.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable react/react-in-jsx-scope */
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import { App } from './App.component';
+import HomePage from '../../pages/Home';
+
+beforeEach(() => {
+  render(<App />);
+  render(<HomePage />);
```

Comment:
In this test suite should be only rendered App, because the App contains `HomePage`component, and fomepage can be test on its own direcoty